README.md

```diff
@@ -68,28 +68,50 @@ and integration tests including pact tests:
 After `npm install`, files will be created under both .git/hooks and .husky, that will automatically lint (and fix) any staged files in your commits, plus run a type check.
 
 ## Running the app
-The easiest way to run the app is to use docker compose to create the service and all dependencies. This starts the latest published docker container for hmpps-auth and redis, a fake manage-recalls-api (wiremock) and a local build of the manage-recalls-ui. 
+In easiest way to run ui and api is to use
 
-`docker compose up`
-
-OR use the following script to run in the background and ensure the fake-mange-recalls-api is up to date and all services start correctly:
+`./start-local-services.sh`
 
-`scripts/start-local.sh` 
+script in manage-recall-e2e project.
 
 Either way check that this has succeeded e.g. via login locally (`http://localhost:3000/`)
 with `PPUD_USER` / `password123456`.  
 This user has the `MANAGE_RECALLS` role that allows access to the service.
 
-### Running the app for development
+### Running the app for the development
+For development run of the ui run
```

Comment:
You don't need to run this in addition to start-local-services.sh as that starts -ui already
You only need to run npm run kill then npm run start:e2e if you want to restart the -ui

Reply:
Yes, in my head it is for development, when we run e2e but after seeing the issue we need to restart it. Any suggestions for rewording it?